### ISSUE TITLE
Reclassify all Land Cover data in drb 

### ISSUE THREAD

**Issue by msleckman** [USGS-R/drb-inland-salinity-ml#55]
Land cover classes in [drb backcasted data](https://www.sciencebase.gov/catalog/item/605c987fd34ec5fa65eb6a74) != land cover classes in [NLCD 06-19](https://www.mrlc.gov/data/legends/national-land-cover-database-2019-nlcd2019-legend). 
Drb backcasted data uses cropland data layer classes (<11) + NLCD data layer classes (>11)

Extraction of drb historical Land Cover metadata below: 
```
Land-cover Classes
Thematic land-cover labels for each pixel
Thematic labels primarily correspond to those from the National Land Cover Database (NLCD)     
0, Corn 
No Data - Area outside of Delaware River Basin boundary 
1, Hay / Alfalfa 
Cropland Data Layer 
2, Corn 
Cropland Data Layer 
3, Soybeans 
Cropland Data Layer 
4, Fallow/Idle 
Cropland Data Layer 
5, Wheat 
Cropland Data Layer 
6, Fruits / Vegetables 
Cropland Data Layer 
7,  Tobacco 
Cropland Data Layer 
8, Other Crops 
Cropland Data Layer 
9, Open Water 
National Land Cover Database 
10, Developed Low Intensity 
National Land Cover Database 
11, Developed Med-High Intensity 
National Land Cover Database 
12, Barren 
National Land Cover Database 
13, Deciduous Forest 
National Land Cover Database 
14, Evergreen Forest 
National Land Cover Database 
15, Mixed Forest 
National Land Cover Database 
16,  Shrubland 
National Land Cover Database 
17, Grassland / Pasture 
National Land Cover Database 
18, Woody Wetlands 
 National Land Cover Database 
 19, Herbaceous Wetland 
 National Land Cover Database 
 20,  Perennial Grasses 
Representing switchgrass, miscanthus, prairie grasses, or other heavily managed grass crops used for cellulosic biofuel feedstock 


```

* Evaluate land cover classes and reclassify + harmonize.

**Comment by jds485** [comment]
The crop classes 0 - 8 can be assigned to NLCD classes 81 (pasture/hay) or 82 (cultivated crops). We can group NLCD medium and high development, as is done here for FORE-SCE. The most unclear match to me is NLCD developed open space. Maybe it's grouped with low intensity?

**Comment by msleckman** [comment]
Reclassification proposal: 

* Reclassified table: [Reclassified_Land_Cover_IS.csv](https://github.com/USGS-R/drb-inland-salinity-ml/files/7903723/Reclassified_Land_Cover_IS.csv)
```
Value,Description 
1,Water
2,Pasture/Hay
3,Cultivated Crops 
4,Developed Areas
5,Barren
6,Forest
7,Shrub
8,Grassland
9,Wetlands
```

* FORESCE legend table with reclassed value matching. table: [Legend_FORESCE_Land_Cover.csv](https://github.com/USGS-R/drb-inland-salinity-ml/files/7903720/Legend_FORESCE_Land_Cover.csv)

```
FORESCE,FORESCE_Description,Reclassify_match,Reclassify_description
0,Corn (no data),3,Cultivated Crops
1,Hay / alfalfa,3,Cultivated Crops
2,Corn,3,Cultivated Crops
3,Soybeans,3,Cultivated Crops
4,Fallow/Idle,2,Pasture/Hay
5,Wheat,3,Cultivated Crops
6,Fruits / Vegetables,3,Cultivated Crops
7,Tobacco,3,Cultivated Crops
8,Other Crops,3,Cultivated Crops
9,Open Water,1,Water
10,Developed Low Intensity,4,Developed Areas
11,Developed Med-High Intensity,4,Developed Areas
12,Barren,5,Barren
13,Deciduous Forest,6,Forest
14,Evergreen Forest,6,Forest
15,Mixed Forest,6,Forest
16,Shrubland,7,Shrub
17,Grassland / Pasture,8,Grassland
18,Woody Wetlands,9,Wetlands
19,Herbaceous Wetlands,9,Wetlands
20,"Perennial grasses - Representing switchgrass, miscanthus, prairie grasses, or other heavily managed grass crops used for cellulosic biofuel feedstock ",2,Pasture/Hay
```

* NLCD11 legend table with reclassed value matching. table (fyi long NLCD descriptors): [Legend_NLCD_land_Cover.csv](https://github.com/USGS-R/drb-inland-salinity-ml/files/7903702/Legend_NLCD_land_Cover.csv)
```
Value,Description,Reclassify_match,Reclassify_description
11,"Open Water - areas of open water, generally with less than 25% cover of vegetation or soil.",1,Water
12,"Perennial Ice/Snow - areas characterized by a perennial cover of ice and/or snow, generally greater than 25% of total cover.",NA,NA
21,"Developed, Open Space - areas with a mixture of some constructed materials, but mostly vegetation in the form of lawn grasses. Impervious surfaces account for less than 20% of total cover. These areas most commonly include large-lot single-family housing units, parks, golf courses, and vegetation planted in developed settings for recreation, erosion control, or aesthetic purposes.",4,Developed Areas
22,"Developed, Low Intensity - areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 20% to 49% percent of total cover. These areas most commonly include single-family housing units.",4,Developed Areas
23,"Developed, Medium Intensity - areas with a mixture of constructed materials and vegetation. Impervious surfaces account for 50% to 79% of the total cover. These areas most commonly include single-family housing units.",4,Developed Areas
24,"Developed High Intensity - highly developed areas where people reside or work in high numbers. Examples include apartment complexes, row houses and commercial/industrial. Impervious surfaces account for 80% to 100% of the total cover.",4,Developed Areas
31,"Barren Land (Rock/Sand/Clay) - areas of bedrock, desert pavement, scarps, talus, slides, volcanic material, glacial debris, sand dunes, strip mines, gravel pits and other accumulations of earthen material. Generally, vegetation accounts for less than 15% of total cover.",5,Barren
41,"Deciduous Forest - areas dominated by trees generally greater than 5 meters tall, and greater than 20% of total vegetation cover. More than 75% of the tree species shed foliage simultaneously in response to seasonal change.",6,Forest
42,"Evergreen Forest - areas dominated by trees generally greater than 5 meters tall, and greater than 20% of total vegetation cover. More than 75% of the tree species maintain their leaves all year. Canopy is never without green foliage.",6,Forest
43,"Mixed Forest - areas dominated by trees generally greater than 5 meters tall, and greater than 20% of total vegetation cover. Neither deciduous nor evergreen species are greater than 75% of total tree cover.",6,Forest
51,"Dwarf Scrub - Alaska only areas dominated by shrubs less than 20 centimeters tall with shrub canopy typically greater than 20% of total vegetation. This type is often co-associated with grasses, sedges, herbs, and non-vascular vegetation.",NA,NA
52,"Shrub/Scrub - areas dominated by shrubs; less than 5 meters tall with shrub canopy typically greater than 20% of total vegetation. This class includes true shrubs, young trees in an early successional stage or trees stunted from environmental conditions.",7,Shrub
71,"Grassland/Herbaceous - areas dominated by gramanoid or herbaceous vegetation, generally greater than 80% of total vegetation. These areas are not subject to intensive management such as tilling, but can be utilized for grazing.",8,Grassland
72,"Sedge/Herbaceous - Alaska only areas dominated by sedges and forbs, generally greater than 80% of total vegetation. This type can occur with significant other grasses or other grass like plants, and includes sedge tundra, and sedge tussock tundra.",NA,NA
73,Lichens - Alaska only areas dominated by fruticose or foliose lichens generally greater than 80% of total vegetation.,NA,NA
74,"Moss - Alaska only areas dominated by mosses, generally greater than 80% of total vegetation.",NA,NA
81,"Pasture/Hay - areas of grasses, legumes, or grass-legume mixtures planted for livestock grazing or the production of seed or hay crops, typically on a perennial cycle. Pasture/hay vegetation accounts for greater than 20% of total vegetation.",2,Pasture/Hay
82,"Cultivated Crops - areas used for the production of annual crops, such as corn, soybeans, vegetables, tobacco, and cotton, and also perennial woody crops such as orchards and vineyards. Crop vegetation accounts for greater than 20% of total vegetation. This class also includes all land being actively tilled.",3,Cultivated Crops
90,Woody Wetlands - areas where forest or shrubland vegetation accounts for greater than 20% of vegetative cover and the soil or substrate is periodically saturated with or covered with water.,9,Wetlands
95,Emergent Herbaceous Wetlands - Areas where perennial herbaceous vegetation accounts for greater than 80% of vegetative cover and the soil or substrate is periodically saturated with or covered with water.,9,Wetlands
```

**Comment by jds485** [comment]
This reclassification looks good! Have you evaluated how much of the DRB is covered by each of your suggested classes? / Viewed a map of the resulting classes? Depending on how much of the area is covered by NLCD open space and low intensity, it could be useful to keep the 2 developed area classes that are in the FORE-SCE classification. NLCD open and low can be in the Developed Low Intensity class.

**Comment by msleckman** [comment]
@jds485  I haven't yet. But I'll create a branch for this LC Class standardization step and check outputs. 
I like the idea of having two developed areas even without graphing as I expect is a clear influence on salt output and the degree of urbanization in an area.

**Comment by msleckman** [comment]
Added related issue #71

**Comment by msleckman** [comment]
Working branch in my forked repo : https://github.com/msleckman/drb-inland-salinity-ml/tree/LandCover_standardization

**Comment by jds485** [comment]
@msleckman: I think this issue could be closed after the NLCD PR #97 is merged. Do you agree? If so, can you link this issue to that PR?
